### ISSUE TITLE
Call to undefined method - WC_Order::is_paid()

### ISSUE THREAD

**Issue by MRokas** [baselbers/woocommerce-pdf-invoices#161]
```
Fatal error: Call to undefined method WC_Order::is_paid() in /var/www/vhosts/test.com/httpdocs/wp-content/plugins/woocommerce-pdf-invoices/includes/abstracts/abstract-bewpi-invoice.php on line 335
```

WC Version is 2.4.0;  
Plugin is latest version;

Any ideas what causes it?

**Comment by MRokas** [comment]
changing `$this->order->is_paid()` to `false` works fine

**Comment by baselbers** [comment]
You are using an older version of WC. Please update to newer version.

**Comment by MRokas** [comment]
Just went through mindfuck-ride. Literally.

1. Plugins page didn't say anything about WooCommerce update being available.
2. Turns out it was other plugin disabling update for it.
3. Updated WooCommerce
4. Everything broke.
5. Reverted back to same version of WooCommerce
6. Still doesn't work
7. Compared ALL files of backup'ed plugin and woocommerce's files from github - TONFUCK of changes directly in plugin.

I think I will need bunch of boxes of beer to fix this monstrous website project I inherited at work....
